packages/core/src/payment/strategies/worldpayaccess/worldpayaccess-payment-strategy.spec.ts

```diff
@@ -262,7 +262,35 @@ describe('WorldpayaccessPaymetStrategy', () => {
                 .toHaveBeenCalledTimes(2);
         })
 
-        it('throws error when url in hidden ifreme is invalid', async () => {
+        it('submit with collection data required hosted supporting firefox', async () => {
+            const createElement = document.createElement;
+            const iframe = document.createElement('iframe')
+            document.body.appendChild(iframe);
+
+            window.document.createElement = jest.fn().mockImplementationOnce(() => iframe).mockImplementation(createElement)
+
+            jest.spyOn(document.body, 'appendChild').mockImplementationOnce(() => undefined )
+
+            if (iframe.contentWindow !== null) {
+                jest.spyOn(iframe.contentWindow.document,'getElementById').mockImplementationOnce(() => undefined)
+            }
+
+            window.fetch = jest.fn(() => Promise.resolve({ok: true}));
+
+            jest.spyOn(orderActionCreator, 'loadCurrentOrder')
+                .mockReturnValueOnce(of(createErrorAction(PaymentActionType.SubmitPaymentFailed, errorResponseAdditionalAction)));
+
+            await strategy.initialize(initializeOptions);
+            await strategy.execute(payload);
+
+            expect(orderActionCreator.loadCurrentOrder).toHaveBeenCalled();
+            expect(form.submit)
+                .toHaveBeenCalledTimes(2);
+
+            window.document.createElement = createElement;
```

Comment:
Do we still need this line after assertions?

Reply:
you was right, it wasn't need it, I removed it